sdk/formrecognizer/ai-form-recognizer/src/formRecognizerClient.ts

```diff
@@ -487,7 +488,7 @@ ng", and "image/tiff";
     }
     const analyzePollerClient: RecognizePollerClient<RecognizeFormResultResponse> = {
       beginRecognize: (
-        body: FormRecognizerRequestBody,
+        body: FormRecognizerRequestBody | string,
```

Comment:
I don't know why yet but the usage of my previous shims masked some compilation errors.